app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss

```diff
@@ -9,11 +9,6 @@
 
 .govspeak, // Legacy class name that's still used in some content items - needs to be kept until `.govspeak` is removed from the content items.
 .gem-c-govspeak {
-  // Scope attachment and attachment-link component styles to gem-c-govspeak
-  @import "../attachment-link";
-  @import "../attachment";
```

Comment:
It looks like the main reason for importing the attachment stylesheets here is to ensure the components are rendered correctly when nested in either a `govspeak` or `gem-c-govspeak` class.

For example, Percy is now showing a [visual change for the attachment component](https://percy.io/55ba5e1a/web/govuk_publishing_components/builds/38950974/changed/2104954930?browser=firefox&browser_ids=65&group_snapshots_by=similar_diff&subcategories=unreviewed%2Cchanges_requested&utm_source=github_status_public&viewLayout=overlay&viewMode=original&width=1280&widths=375%2C1280), essentially reverting the change made in  https://github.com/alphagov/govuk_publishing_components/pull/856

Reply:
Thanks for spotting this @MartinJJones. I think I can mitigate this by adding these extra styles into the attachment component sass file. This isn't ideal but I think it might still be better than duplicating all this Sass. There's still a slight visual diff as the last paragraph now has an override bottom margin that isn't then overridden by govspeak (it would normally take the bottom margin off the last paragraph) - but I think that's about to change anyway in https://github.com/alphagov/govuk_publishing_components/pull/4623

Reply:
This approach makes sense to me, the fix looks good. As you say https://github.com/alphagov/govuk_publishing_components/pull/4623 seems to be a quite a big update to spacing, agree it would make sense to get this change in first and then look at it again in the spacing updates PR.